static/src/javascripts/projects/common/modules/audio/AudioPlayer.js

```diff
@@ -324,6 +323,8 @@ export default class AudioPlayer extends Component<Props, State> {
         this.dataArray = new Uint8Array(this.analyser.frequencyBinCount);
         this.source = this.context.createMediaElementSource(this.audio);
         this.source.connect(this.analyser);
+        this.analyser.connect(this.context.destination);
+        this.audio.crossOrigin = 'anonymous';
```

Comment:
This line is not necessary to solve the bug, but will be necessary to set up the CORS headers properly